step1/vanila-todo/TodoController.ts

```diff
@@ -0,0 +1,72 @@
+import TodoModel from "./TodoModel";
+import TodoView from "./TodoView";
+
+export default class TodoController {
+  model: TodoModel;
+  view: TodoView;
+
+  constructor(model: TodoModel) {
+    this.model = model;
+    this.view = model.view;
+  }
```

Comment:
```suggestion
  constructor(public readonly model: TodoModel) {}
  get view () { return this.model.view }
```
이렇게 표현할 수 있을 것 같네요!
view를 굳이 멤버변수로 관리할 필요는 없을 것 같아요.
그리고 생성자에서 저렇게 바로 접근제한자를 지정해주면 바로 멤버변수에 할당된답니다 ㅎㅎ